packages/syft/src/syft/client/client.py

```diff
@@ -768,7 +782,10 @@ def login(
 
     if login_credentials is None:
         if verbose:
-            print(f"Logged into {_client.name} as GUEST")
+            print(
+                f"Logged into {_client.name} {_client.metadata.node_side_type} "
```

Comment:
Could we have separation with angle brackets, to make it clear

like
"Logged into < {_client.name} - {_client.metadata.node_type} > - {_client.metadata.node_side_type} side as GUEST"

or any other nice way to make it coherent.

Reply:
When I tried it , it was hard to distinguish without separation.

Reply:
fixed: https://github.com/OpenMined/PySyft/pull/7916/commits/e3dbf0b80c65ebe49c6791faad1b542d73ba9381